libcxx/include/__chrono/zoned_time.h

```diff
@@ -147,8 +148,29 @@ class zoned_time {
     } && is_convertible_v<sys_time<_Duration2>, sys_time<_Duration>>)
       : zoned_time{__traits::locate_zone(__name), __zt, __c} {}
 
+  _LIBCPP_HIDE_FROM_ABI zoned_time& operator=(const sys_time<_Duration>& __tp) {
+    __tp_ = __tp;
+    return *this;
+  }
+
+  _LIBCPP_HIDE_FROM_ABI zoned_time& operator=(const local_time<_Duration>& __tp) {
+    // TODO TZDB This seems wrong.
+    // Assigning a non-existant or abiguous time will throw and not satisfy
```

Comment:
```suggestion
    // Assigning a non-existent or ambiguous time will throw and not satisfy
```

Can you file a LWG issue about this?

Reply:
As mentioned I like to write a paper for all the issues found instead of a set of LWG issues.